Grand Circus - OOP/GrandCircus/CircusModel/AnimalBase.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal class AnimalBase
```

Comment:
Because AnimalBase is only used as a base class, we need to make it 'abstract': internal abstract class AnimalBase: IAnimal. 
https://www.geeksforgeeks.org/c-sharp-abstract-classes/